playwright_tests/fixtures.ts

```diff
@@ -0,0 +1,84 @@
+import { BrowserContext } from '@playwright/test';
+import { test as base, expect } from 'playwright-test-coverage';
+import { cookieCategories } from '../src/lib/cookies';
+
+export class CookieConsentFixture {
```

Comment:
should probably remove the cookie consent stuff as the component is not used in the template